ch1/study.css

```
@@ -0,0 +1,5 @@
+/* */
+div#main-item {
+    margin: 0 auto;
+    width: 1216px;
+}
```

```suggestion
div#main-item {
  margin: 0 auto;
  width: 1216px;
}
```

CSS 역시 마찬가지로 들여쓰기 2칸 사용합니다.